assembl/models/timeline.py

```diff
@@ -44,12 +45,26 @@ class TimelineEvent(DiscussionBoundBase):
         'with_polymorphic': '*'
     }
 
-    title = Column(CoerceUnicode(), nullable=False,
+    title_id = Column(Integer(), ForeignKey(LangString.id), nullable=False,
         info={'rdf': QuadMapPatternS(None, DCTERMS.title)})
 
-    description = Column(UnicodeText,
+    description_id = Column(Integer(), ForeignKey(LangString.id),
         info={'rdf': QuadMapPatternS(None, DCTERMS.description)})
 
+    title = relationship(LangString,
+        lazy="joined", single_parent=True,
+        primaryjoin=title_id == LangString.id,
+        backref=backref("title_of_timeline_event", lazy="dynamic"),
+        cascade="all, delete-orphan")
+
+    description = relationship(LangString,
+        lazy="joined", single_parent=True,
```

Comment:
joined: I want the description to load with the timeline.
dynamic: I want to update the langstring without having to traverse back to the timeline unless necessary.